addons/addon-base/packages/serverless-go-build-tools/index.js

```diff
@@ -71,6 +75,15 @@ class GoBuildTools {
     });
   }
 
+  // From: https://stackoverflow.com/questions/12042534/node-js-synchronous-prompt/46700053#46700053
```

Comment:
Instead of a link, could you describe what this method does as a one-liner?

Reply:
Included both!